WebDevStudios/nextjs-wordpress-starter
# Feature/828 fullhead

Closes #828

### Description

Fixes links in docs
Updates SEO handling to use Yoast `fullHead`

### Screenshot

![Screen Shot 2022-01-04 at 3 34 12 PM](https://user-images.githubusercontent.com/36422618/148133161-221adaff-32d5-40a2-b508-8ee2adc3a1e0.png)

### Verification

1. `gh pr checkout 846`
2. `cd docs`
3. `npm run build && npm run start`
4. `cd ../frontend`
5. `npm run build && npm run start`
6. check Log In, Register, Profile, Search pages for manual SEO meta
7. check various pages/posts for dynamic SEO meta
